go/vt/vttablet/tabletmanager/restore.go

```diff
@@ -44,6 +49,13 @@ var (
 	restoreFromBackup     = flag.Bool("restore_from_backup", false, "(init restore parameter) will check BackupStorage for a recent backup at startup and start there")
 	restoreConcurrency    = flag.Int("restore_concurrency", 4, "(init restore parameter) how many concurrent files to restore at once")
 	waitForBackupInterval = flag.Duration("wait_for_backup_interval", 0, "(init restore parameter) if this is greater than 0, instead of starting up empty when no backups are found, keep checking at this interval for a backup to appear")
+
+	// Flags for PITR
+	restoreToTimeStr = flag.String("restore_to_time", "", "(init restore parameter) will restore to the specified time, it depends on the binlog related flags.")
```

Comment:
are we requiring that the timestamp set in the recovery keyspace and this time should match?
what happens if someone provides this though they are not actually doing a PITR?

Reply:
ohh I forgot, I though snapshot time = backup time, I will change my code.